### ISSUE TITLE
`make local-iso` doesn't find overlayfs/boot/vmlinuz

### ISSUE THREAD

**Issue by kkaempf** [rancher-sandbox/cOS-toolkit#293]
**cos-toolkit version:**

git master @ d1c26114b4ae98de2b8060e7ed2662079416719f 

**CPU architecture, OS, and Version:**

Tumbleweed x86_64

**Describe the bug**

`sudo make build` followed by `make local-iso` fails with
```
ERRO[0000] fatal error: stat /tmp/luet-geniso118758811/overlayfs/boot/vmlinuz: no such file or directory
```

**To Reproduce**

**Expected behavior**

**Logs**

This is suspicious:

```
INFO[0000] 🍜  Installing Overlay packages               
 Downloading file repository.yaml from /abuild/projects/rancher/cOS/cOS-toolkit/build
 Downloading file tree.tar.zst from /abuild/projects/rancher/cOS/cOS-toolkit/build
 Downloading file repository.meta.yaml.tar.zst from /abuild/projects/rancher/cOS/cOS-toolkit/build
 🏠  Repository cOS revision: 1 - 2021-06-17 15:54:47 +0200 CEST
 ℹ Repository: local Priority: 0 Type: disk
 No packages to install
 Cleaned:  0 packages.
```

"No packages to install" ?!

**Additional context**

**Comment by mudler** [comment]
Need full logs of the build too, it looks like nothing was actually built

**Comment by kkaempf** [comment]
Hmm, I can't easily reproduce this bug. However, I do remember that Darren reported this problem too.

Trying to find a reproducer ...

**Comment by mudler** [comment]
In the specific case of Darren, the build failed leaving half packages built as it failed building SELinux packages, I'm afraid this is the same case, where the build didn't actually succeeded. We should check when this is happening the content of the build folder

**Comment by kkaempf** [comment]
Apparently this happens with unprivileged `make` runs. Similar as #143 (fixed by #146)

**Comment by kkaempf** [comment]
Fixed by #146
